dothething/dothething/Views/ProfileView.swift

```diff
@@ -264,7 +264,7 @@ extension ProfileView {
                             // save url to clips array
                             DispatchQueue.main.async {
                                 let thumbnail = Thinger.getThumbnail(url: tempFileUrl, degreesToRotate: self.videoDegreesToRotate)
-                                self.clips.append(Clip(url: tempFileUrl, thumbnail: thumbnail, isHighlighted: false, metadata: clip))
+                                self.clips.append(Clip(url: tempFileUrl, thumbnail: thumbnail, isHighlighted: false, metadata: clip, showCode: true))
```

Comment:
The change to the initialization of `Clip` instances within the `ProfileViewModel` is also consistent with the previous changes, ensuring that the `showCode` property is set correctly for each clip.